### DISCUSSION TITLE
problem with the page url

### DISCUSSION THREAD

**Discussion by kamyarothmanhamad** [community/community#114018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have created a repository named (kamyaro.github.io) to make a page for myself. the page is live now but, the problem is the url 
contains my github name as well meaning the github is live with this url https://kamyarothmanhamad.github.io/kamyaro.github.io/ instead of kamyaro.github.io

**Comment by PATRICKNDIZEYE** [comment]
To host your GitHub Pages site directly at kamyaro.github.io, rename your repository to kamyaro.github.io in the repository settings on GitHub. Optionally, update the Git remote URL if you have cloned the repository locally. Wait a few minutes for GitHub Pages to reflect the changes, and your site should be accessible at https://kamyaro.github.io.

**Reply by kamyarothmanhamad** [reply]
the thing is, my github name was kamyaro.github.io at the first place. but still the problem persist. check out the attached image please.
![Capture](https://github.com/community/community/assets/97412888/19239922-e4b0-4111-ad91-ce076183bf1a)

**Reply by PATRICKNDIZEYE** [reply]
check repository settings ,Verify DNS Configuration ,  Clear Browser Cache , Wait for DNS Propagation

**Comment by Kakar21** [comment]
I would just copy everything, delete the repository and create a new one with the domain you like and paste your stuff in there.

**Reply by kamyarothmanhamad** [reply]
i have done this a few times. still facing the same problem!
